src/OSPSuite.InstallationValidator.Core/Presentation/DTO/FolderDTO.cs

```diff
@@ -0,0 +1,48 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Presentation.DTO;
+using OSPSuite.Utility;
+using OSPSuite.Utility.Validation;
+
+namespace OSPSuite.InstallationValidator.Core.Presentation.DTO
+{
+   public class FolderDTO : ValidatableDTO
+   {
+      private string _targetFolder;
```

Comment:
Rename to _folder maybe? _target also implies in general a Source but we do not have that here